timeeval/adapters/distributed.py

```diff
@@ -9,7 +9,24 @@
 
 class DistributedAdapter(Adapter):
     """
+    An adapter that allows to run a function as an anomaly detector on multiple remote machines. So far, this adapter
+    only supports TSFunctions as algorithms.
     Please, be aware that you need password-less ssh to the remote machines!
+
```

Comment:
Please add a warning that this adapter is deprecated and does not support all features of TimeEval!

Warnings can be created via:

```rst
.. warning::
    Text
```